MUXSDKKaltura/MUXSDKKalturaTests/MUXSDKStatsTests+Utilities.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  MUXSDKStatsTests+Utilities.swift
+//  MUXSDKKalturaTests
+//
+//  Created by Audra Rodriguez on 18/11/21.
+//  Copyright © 2021 Mux, Inc. All rights reserved.
+//
+
+import XCTest
+@testable import MUXSDKKaltura
+import MuxCore
+
+extension MUXSDKStatsTests {
+    private enum EventType{
+        case customerVideoData
+        case customerPlayerData
+        case customerData
+        case customData
+    }
+    
+    func assertDispatchedEventTypesForPlayer(id: String, expectedEventTypes: [String]) {
```

Comment:
these assert methods need more clear names as it's not totally clear what they are doing. i'm guessing what you are trying to do here is assert that the events **match** the player, so you should be including the words "equal to" or "match" in there

eg
` assertDispatchedEventTypesMatch(expectedEventTypes: [String], for playerId: String)`
or even alternatively, you could just do get rid of this function entirely and add a utility for getting the events for a player and do something like `XCTAssertEqual(eventsForPlayer(playerId).map { $0.getType() }, expectedEvents)`